packages/Alert/src/Alert.tsx

```diff
@@ -1,178 +1,186 @@
-    <IconButton
-      appearance={{ type: 'ghost', variant: 'secondary' }}
-      className="ids-alert__dismiss-btn"
-      type="button"
-      size="medium"
-      icon={<Close size="medium" />}
-      onClick={action}
-    />
-  );
+    return (
+        <IconButton
+            appearance={{ type: "ghost", variant: "secondary" }}
+            className="ids-alert__dismiss-btn"
+            type="button"
+            size="medium"
+            icon={<Close size="medium" />}
+            onClick={action}
+        />
+    );
 };
 
 const renderAlertActionButton = (
-  style: Appearance,
-  button?: AlertButton,
+    style: Appearance,
+    button?: AlertButton
 ): JSX.Element => {
-  if (button == null || button.onClick == null || button.label == null) {
-    return <></>;
-  }
-
-  return (
-    <Button
-      appearance={style === 'horizontal' ? 'ghost' : 'secondary'}
-      size="small"
-      onClick={button.onClick}
-      className="ids-alert__action-btn"
-    >
-      {button.label}
-    </Button>
-  );
+    if (button == null || button.onClick == null || button.label == null) {
+        return <></>;
+    }
+
+    const ghostAppearance = getBrand() === "workleap" ? 
+        { type: "ghost", variant: "secondary" } as ButtonAppearance : 
+        "ghost";
+
+    return (
+        <Button
+            appearance={style === "horizontal" ? ghostAppearance : "secondary"}
+            size="small"
+            onClick={button.onClick}
+            className="ids-alert__action-btn"
+        >
+            {button.label}
+        </Button>
+    );
 };
 
 const Alert: React.FunctionComponent<AlertProps> = ({
-  title,
-  message,
-  metadata,
-  type,
-  appearance = 'card',
-  className,
-  closable = true,
-  icon,
-  onClose,
-  button,
-  dataTest,
-  ...rest
-}: AlertProps) => {
-  const classes = classNames(
-    'ids-alert',
-    `ids-alert--${appearance}`,
-    `ids-alert--${type}`,
+    title,
+    message,
+    metadata,
+    type,
+    appearance = "card",
     className,
-  );
-
-  const parentElement = React.useRef<HTMLDivElement>(null);
-  const [show, setShow] = React.useState(true);
-  const hasButton = button !== undefined;
-  const isHorizontal = appearance === 'horizontal';
-  const canBeClose = closable && !isHorizontal;
+    closable = true,
+    icon,
+    onClose,
+    button,
+    dataTest,
+    ...rest
+}: AlertProps) => {
+    const classes = classNames(
+        "ids-alert",
+        `ids-alert--${appearance}`,
+        `ids-alert--${type}`,
+        className
+    );
 
-  if (show) {
-    return (
-      <div
-        className={classes}
-        ref={parentElement}
-        data-test={dataTest}
-        {...rest}
-      >
-        {icon !== null &&
+    const parentElement = React.useRef<HTMLDivElement>(null);
+    const [show, setShow] = React.useState(true);
+    const hasButton = button !== undefined;
+    const isHorizontal = appearance === "horizontal";
+    const canBeClose = closable && !isHorizontal;
```

Comment:
isDismissible? that would follow the isHorizontal convention

Reply:
same as above, it's not part of what changed